tensorflow/lite/micro/kernels/BUILD

```diff
@@ -317,6 +317,7 @@ tflm_kernel_cc_library(
         "//tensorflow/lite/micro:flatbuffer_utils",
         "//tensorflow/lite/micro:memory_helpers",
         "//tensorflow/lite/micro:micro_graph",
+        "//tensorflow/lite/micro:micro_log",
```

Comment:
I do not see any Kernel file changes related to micro_log in this PR. Does this dependency required?

Reply:
This is needed because the kernels are using MicroPrintf